Update docs/README.Windows.md

thexai opened the pull request:
## Description
Update docs/README.Windows.md

## Motivation and context
- Reflect Visual Studio 2019 install notes
- Update latest CMake requirements (v3.15) since https://github.com/xbmc/xbmc/pull/19315 merged
- Update command line examples.

## How has this been tested?
<!--- Please describe in detail how you tested your change. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## What is the effect on users?
<!--- Summarize the effect of this change on Kodi end-users. -->
<!--- If the PR does not have a noticeable impact (e.g., if it only changes documentation), -->
<!--- just leave it empty. Put in more detail the bigger the impact is. -->
<!--- This section may be used for automatic creation of release notes. -->

## Screenshots (if appropriate):

## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [X] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
<!--- Go over all the following points, and put an `X` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

thexai committed:
Update docs/README.Windows.md

garbear commented on docs/README.Windows.md:
```diff
@@ -46,11 +46,11 @@ Several different strategies are used to draw your attention to certain pieces o
 ## 2. Prerequisites
 To build Kodi:
-* **[CMake](https://cmake.org/download/)**
+* **[CMake](https://cmake.org/download/)** (is required version 3.15 or greater)
```
"version 3.15 or greater is required" or ""requires version 3.15 or greater"

garbear reviewed (approved):
looks good other than the minor grammar suggestion

thexai commented:
Added  `MSVC v142 - VS 2019 C++ ARM build tools (Latest)`  in individual components. 
Is the only component that needs to be manual added to compile UWP-ARM variant (when "Universal Windows Platform Development" and SDK is already checked).

Added a note to explain the process of build `peripheral.joystick` only add-on and the installer including it same as official Windows installers.
